### PR TITLE
Restructure of getting started 

### PR DESCRIPTION
**Pull request by CaroMac**

### CODE REVIEW

File: src/markdown-pages/docs/first-steps/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+path: "/docs/prerequisites"
```
Comment (jt-nti) [inline src/markdown-pages/docs/first-steps/prerequisites.md:2]: I can't see this page in the contents, or any way to get to it.

File: src/markdown-pages/docs/first-steps/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+
+### <a name="gradle"></a>Gradle
+
+<a href="https://docs.gradle.org" target="_blank">Gradle</a> is required if you are installing the zipped distribution of Galasa, or if you want to use Gradle rather than Maven to build Galasa projects (hierarchical file structures that provide the ability to store and run Galasa tests).
+
+The following table shows the current compatibility between Gradle and Galasa versions: 
+
+
+| Gradle release |  Compatible Galasa version  |
```
Comment (jt-nti) [inline src/markdown-pages/docs/first-steps/prerequisites.md:60]: Is a table required? It looks like Galasa supports Gradle 6.8.x, 6.9.x and 7.x.x.

File: src/markdown-pages/doc_root.md [hunk 3]
```diff
@@ -1,45 +1,23 @@
...
-This hub page contains links to the main Galasa documentation. To get going with Galasa as quickly as possible, explore the [Getting started](/docs/getting-started) section. Read on to learn a little about Galasa's architecture first.
 
-# Galasa's architecture
-At its top level, Galasa decomposes into three major components:
+There are a few options available when installing Galasa. Prerequisites vary, depending on the option that is chosen. 
 
-* The underlying core Galasa framework
-* A collection of Managers 
-* A test runner
+You can install Galasa for using in the command-line, for using Eclipse, or for sharing with your department (using the Galasa zipped distribution).
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/doc_root.md:8]: Thank you - have updated

File: src/markdown-pages/docs/architecture.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+path: "/docs/architecture"
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/architecture.md:2]: thanks James - I will have a think about that.

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+| Operating system  |  Download  |
+| :---- | :-------- | 
+| MacOSX  | galasactl-darwin-x86_64 |
+| MacOSX  | galasactl-darwin-arm64 |
+| Linux 64-bit x86 | galasactl-linux-x86_64 | 
+| Linux arm64 | galasactl-linux-arm64 | 
+| zLinux  | galasactl-linux-s390x | 
+| Windows | galasactl-windows-x86_64.exe | 
+
+You can find out the architecture of your machine by typing the command `uname -m` into your Mac or Linux terminal.
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:32]: yes, I see what you are saying. I was thinking someone might wonder which to download before reviewing the install steps but it does then read like a duplicate so I will remove the top one and keep it in the Mac specific steps.

File: src/markdown-pages/docs/first-steps/initialising-home-folder.md [hunk 6]
```diff
@@ -52,42 +60,40 @@ ${HOME}/.galasa
...
+
+If you are planning to only run tests locally, with everything running on the local machine, you do not need to set the Galasa bootstrap. Otherwise, you can set the bootstrap either by using the `--bootstrap` option on the CLI command or by setting the `GALASA_BOOTSTRAP` environment variable. If both are provided, the `--bootstrap` option takes precedence.
+
+You can set environment variables on a terminal by using the `export` (if you are on Mac or Linux) or `set` (if you are on Windows) command. For example, to set `GALASA_BOOTSTRAP` to `http://my-bootstrap-url`, use the following command:
 
 On Mac or Unix:
 
 ``` 
-export GALASA_HOME=/temp/mygalasatests
+export GALASA_BOOTSTRAP=http://my-bootstrap-url
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/first-steps/initialising-home-folder.md:88]: I see, yes will update

File: src/markdown-pages/docs/first-steps/initialising-home-folder.md [hunk 7]
```diff
@@ -52,42 +60,40 @@ ${HOME}/.galasa
...
 ```
 
 On Windows: 
 
 ``` 
-set GALASA_HOME=C:\temp\mygalasatests
+set GALASA_BOOTSTRAP=http://my-bootstrap-url
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/first-steps/initialising-home-folder.md:94]: yes, have updated all 3 instances thank you

File: src/markdown-pages/docs/first-steps/prerequisites.md (hunk 1 above)
Comment (CaroMac) [reply src/markdown-pages/docs/first-steps/prerequisites.md:2]: ah yes I put the pre-reqs in each section as they were slightly different so it made better sense to separate them. I did not delete this file though so have done that now.

File: src/markdown-pages/docs/first-steps/prerequisites.md (hunk 2 above)
Comment (CaroMac) [reply src/markdown-pages/docs/first-steps/prerequisites.md:60]: Yes, I changed that when I moved the prereqs  - this is in the deleted file. thank you!

File: src/markdown-pages/docs/first-steps/initialising-home-folder.md [hunk 8]
```diff
@@ -1,33 +1,41 @@
...
 On Windows: 
 
 ``` 
-set GALASA_BOOTSTRAP=http://my-bootstrap-url
+set GALASA_HOME=C:\temp\mygalasatests
 ```
 
+You can override the value of the `GALASA_HOME` environment variable on a call-by-call basis by using the `--galasahome` command-line option.
+
+If you change the `GALASA_HOME` variable to a non-existent or non-initialised folder path, you can create the folder and re-initialise the folder path by running the `galasactl local init` command.
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/first-steps/initialising-home-folder.md:36]: will check and update

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+4. Set execute permission on the binary by running the `chmod +x galasactl` command in the directory containing `galasactl`.
+5. If you are using a Mac, you can set permission to open the Galasa CLI tool by running the `spctl --add galasactl` command in the directory containing `galasactl`. You are prompted by a security panel asking you to log in to show that you are issuing the command.
+
+You can now run the Galasa CLI too from any directory in your file system without having to specify the absolute path.
+
+
+On Windows (Powershell)
+
+1. Download the binary and re-name it to `galasactl`.
+2. Add `galasactl` to your PATH to enable the tool to be called from the command line without having to specify the path to the directory in which it is stored.
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:53]: have added in extra info for windows

File: src/markdown-pages/docs/first-steps/initialising-home-folder.md [hunk 10]
```diff
@@ -96,4 +102,9 @@ The `settings.xml` file enables you to set configurations for Maven to use durin
 If you have Maven installed, and have run a Maven command, an `/.m2` folder, containing a folder called `repositories`, should exist in your home directory. Running the `galasa local init` command then automatically creates a `settings.xml` file in this `/.m2` folder, and populates the file with basic configuration settings.
 
 
+## Next steps
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/first-steps/initialising-home-folder.md:105]: spoke to Eamonn about this and will make an update

File: src/markdown-pages/docs/first-steps/initialising-home-folder.md [hunk 11]
```diff
@@ -1,33 +1,41 @@
 ---
 path: "/docs/initialising-home-folder"
-title: "Initialising your local environment by using the CLI"
+title: "Initialising your local environment"
 ---
 
 To start using Galasa tools, or running Galasa tests, you need to set up some basic file structures and files in your home folder. These files include a default bootstrap file, and some local properties files. For more information about these files and what they are used for, see the [About the properties files](#about) section. 
 
-The following section shows you how to initialise your Galasa home folder by using the Galasa command line interface (Galasa CLI) tool that is provided with Galasa. Once your home folder is initialised, you can start running Galasa tests on your local JVM. 
+The following section shows you how to initialise your Galasa home folder by using the Galasa command line interface tool (Galasa CLI) that is provided with Galasa. Once your home folder is initialised, you can start running Galasa tests on your local JVM.
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/first-steps/initialising-home-folder.md:8]: Ah yes, I have been through and updated a few instances for consistency

File: src/markdown-pages/docs/first-steps/initialising-home-folder.md [hunk 12]
```diff
@@ -37,7 +45,7 @@ galasactl local init
 ```
 
 
-This one-time command is run once per user and helps you to quickly create the folder and files that you need. If you already have a Galasa home folder set up, the folder and files are not created. 
+This one-time command is run once Galasa home directory and helps you to quickly create the folder and files that you need. If you already have a Galasa home folder set up, the folder and files are not created.
```
Comment (jt-nti) [inline src/markdown-pages/docs/first-steps/initialising-home-folder.md:48]: I think there's a word missing? "per/for each Galasa home directory"?

Comment (CaroMac) [reply src/markdown-pages/docs/first-steps/initialising-home-folder.md:48]: oops thank you! :)

**Review by jt-nti — APPROVED** [review]
LGTM!

**Review by techcobweb — APPROVED** [review]
approving on the basis James is happy

**Pull request merged**